src/test/java/com/ericsson/gerrit/plugins/eiffel/storage/EventStorageTest.java

```diff
@@ -0,0 +1,115 @@
+package com.ericsson.gerrit.plugins.eiffel.storage;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.net.ConnectException;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.rules.TemporaryFolder;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import com.ericsson.gerrit.plugins.eiffel.exceptions.NoSuchElementException;
+import com.ericsson.gerrit.plugins.eiffel.handlers.DataBaseHandler;
+import com.ericsson.gerrit.plugins.eiffel.handlers.Table;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest(fullyQualifiedNames = "com.ericsson.gerrit.plugins.eiffel.*")
+public class EventStorageTest {
+
+    private static final String PROJECT = "project_test";
+    private static final String BRANCH = "branch_test";
+    private DataBaseHandler dbHandler;
+    private SourceChangeCreatedStorage sourceChangeCreatedState;
+    private SourceChangeSubmittedStorage sourceChangeSubmittedState;
+
+    @Rule
+    public TemporaryFolder testFolder = new TemporaryFolder();
+    private File tmpFolderPath;
+
+    @Rule
+    public final ExpectedException exception = ExpectedException.none();
+
+
+    @Before
+    public void init() throws Exception {
+        tmpFolderPath = testFolder.newFolder();
+        dbHandler = Mockito.mock(DataBaseHandler.class);
+        PowerMockito.whenNew(DataBaseHandler.class).withParameterTypes(File.class, String.class).withArguments(Mockito.any())
```

Comment:
Since you use both PowerMockito and Mockito it is good that you write out what you use. I like this, well done. :thumbsup: